pkg/stream/client.go

```diff
@@ -413,10 +413,12 @@ func (c *Client) DeleteStream(streamName string) error {
 }
 
 func (c *Client) heartBeat() {
-	ticker := time.NewTicker(60 * time.Second)
-	tickerHeatBeat := time.NewTicker(20 * time.Second)
+	ticker := time.NewTicker(time.Duration(c.tuneState.requestedHeartbeat) * time.Second)
+	tickerHeatBeat := time.NewTicker(time.Duration(c.tuneState.requestedHeartbeat-2) * time.Second)
```

Comment:
I guess this is a typo
```suggestion
	tickerHeartbeat := time.NewTicker(time.Duration(c.tuneState.requestedHeartbeat-2) * time.Second)
```

And also, just to understand, why it is needed the `-2` here?  `time.Duration(c.tuneState.requestedHeartbeat-2)`
Couldn't be just? `time.Duration(c.tuneState.requestedHeartbeat)` as above?

Reply:
There are two tickers:
1. `ticker`
2. `tickerHeatBeat` 
  

The 1 sends the heartbeat. 
The 2 checks if the heartbeat is received 

To avoid an overlap in timing the 2 checks more frequently. It is just a precaution.

Reply:
Okay, it makes sense.
Since we are here, we can just fix the typo and it is mergeable!